packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -940,7 +942,9 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
    * OnBlur handler. Set the focused state to false
    * and submit any pending value
    */
-  private _onBlur = (event: React.FocusEvent<HTMLInputElement>): void => {
+  private _onBlur = (
+    event: React.FocusEvent<HTMLInputElement | Autofill | HTMLAnchorElement | HTMLButtonElement | HTMLDivElement | BaseButton | Button>
```

Comment:
Is there a better way to fix type for the method?

Reply:
You could widen the type to `React.FocusEvent<HTMLElement>`

Reply:
Trouble is that Autofill, BaseButton, and Button don't extend HTMLElement (I've been running into this same issue in other places)... In general I like having accurate typings, but in this case it seems rather obnoxious.

Reply:
Yeah that's the problem, they all don't extend from `HTMLElement`. Either I could do what I have or rely on `any`.

Reply:
Sure, but the `HTMLElement` sub-classes could have be clumped under `HTMLElement` unioned with the non-`HTMLElement` types. Doesn't matter to me tho.